Contracts/v1.0/buyer/ad-details/error-response.json5

```diff
@@ -0,0 +1,23 @@
+
+//Error response object
+{
+    "code": "401",
+    "message": "You must login first",
+    "info": [
+      {
+      "code": "401",
+      "message": "You must login first"
+      }
+   ]
+ 
+    
+    "code": "404",
```

Comment:
why do we need multiple error response ? if so we need it can we make it as an array of error response?